File: level1JS/numOfPY.js
function solution(s) {
    let p = 0;
    let y = 0;
    s.toLowerCase()
        .replace(/[^py]/g, "")
        .split("")
        .forEach(str => {
            str === "p" ? p++ : y++;
        })
    return p === y;
}

console.log(solution("pPoooyY"))

File: level1JAVA/src/NumOfPY.java
class NumOfPY {
    boolean solution(String s) {
        return s.replaceAll("[^pP]", "").length() == s.replaceAll("[^yY]", "").length();
    }

    public static void main(String[] args) {
        NumOfPY numOfPY = new NumOfPY();
        boolean result = numOfPY.solution("pPoooyY");
        System.out.println("result = " + result);
    }
}
